pkgs/top-level/all-packages.nix

```diff
@@ -14963,6 +14963,8 @@ in
   # linux only by now
   synergy = callPackage ../applications/misc/synergy { };
 
+  systemtap = callPackage ../development/tools/profiling/systemtap { };
+
```

Comment:
You need to place this inside the `linuxPackagesFor` function so it can be built on other kernels than the default `linuxPackages`.